### ISSUE TITLE
Vary amount of training data used in the validation set

### ISSUE THREAD

**Issue by radka-j** [alan-turing-institute/ARC-MTQE#42]
At the moment COMET randomly selects 1000 training records to also evaluate as part of the validation step.

We might want to increase this size if we want to use the training data for picking a binarisation threshold.
